<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>management_GUIClass</class>
 <widget class="QMainWindow" name="management_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>management_GUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 119, 179);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="turret_list_widget">
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(132, 193, 237);
</string>
      </property>
      <property name="lineWidth">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>8514oem</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(241, 241, 241);
font: 8pt &quot;8514oem&quot;;
</string>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="location_line">
        <property name="font">
         <font>
          <family>8514oem</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(132, 193, 237);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>8514oem</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(241, 241, 241);
font: 8pt &quot;8514oem&quot;;</string>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="size_line">
        <property name="font">
         <font>
          <family>8514oem</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(132, 193, 237);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>8514oem</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(241, 241, 241);
font: 8pt &quot;8514oem&quot;;</string>
        </property>
        <property name="text">
         <string>Aura Level</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="aura_level_line">
        <property name="font">
         <font>
          <family>8514oem</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(132, 193, 237);
selection-color: rgb(47, 191, 239);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>8514oem</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(241, 241, 241);
font: 8pt &quot;8514oem&quot;;</string>
        </property>
        <property name="text">
         <string>Parts</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="parts_line">
        <property name="font">
         <font>
          <family>8514oem</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(132, 193, 237);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>8514oem</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(241, 241, 241);
font: 8pt &quot;8514oem&quot;;</string>
        </property>
        <property name="text">
         <string>Vision</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="vision_line">
        <property name="font">
         <font>
          <family>8514oem</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(132, 193, 237);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(140, 228, 237);
font: 8pt &quot;8514oem&quot;;

</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(140, 228, 237);
font: 8pt &quot;8514oem&quot;;</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(140, 228, 237);
font: 8pt &quot;8514oem&quot;;</string>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="exit_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(140, 228, 237);
font: 8pt &quot;8514oem&quot;;</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>turret_list_widget</zorder>
   <zorder></zorder>
   <zorder>exit_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="management_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
